### ISSUE TITLE
Detailslist: selection mode single , narrator announces incorrect column count

### ISSUE THREAD

**Issue by arunarokk** [OfficeDev/office-ui-fabric-react#5747]
### Bug Report 
 
- __Package version(s)_: 6.38.0
- __Browser and OS versions__: (Edge win 10)
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (No)
 
Requested priority: (High) 
 
Products/sites affected: (if applicable)
 
#### Actual behavior:
 
Detailslist, when selection mode is single, Narrator is on in scan mode, 
it announces correct total column count, but the when reading each column header it gives the wrong count. the aria-colindex is started with 2 for the first column header,. this is causing the mismatch.
![selectionmodesingle](https://user-images.githubusercontent.com/38989707/43489874-4b236764-94d3-11e8-9710-5c4a54bcffe8.PNG)

In the above list , for the column "shape" narrator announces, column 7 of 6.

 
https://developer.microsoft.com/en-us/fabric#/components/detailslist

**Comment by KevinTCoughlin** [comment]
Assigning myself in case I can patch this quickly. I'll need to check the ARIA spec because there _is_ a column header there for the "Select All" toggle in both selection modes. We don't allow you to use it in single selection mode, but there is still the column of individual select toggles (per row). So the column header count seems accurate to me, but perhaps we need to announce that the first column is disabled in single selection mode to address the accessibility issue.

![screengrab showing first column toggle visible](https://user-images.githubusercontent.com/706967/44179834-0301bf80-a0ae-11e8-9e9d-b56b009873d6.png)

Thoughts @arunarokk?

**Comment by KevinTCoughlin** [comment]
Ah right, I think there's already a fix out for this that's been merged #5925 🎉. Still wonder if we should use `aria-hidden` but out of scope of this issue.

**Comment by natalieethell** [comment]
Thanks for tagging me on this @KevinTCoughlin! Yes, my PR you linked above should fix this issue, so I'll go ahead and close this. 

Thomas and I decided that it makes sense to maintain that column index for the "select all" checkbox when in single selection mode and non-focusable, because it is still a column header, although it alone is not actionable. So, it should be part of column count. 

I do agree that we should add `aria-hidden` to the "select all" column header in single selection mode. I can send out a PR for that today.
